app/src/main/java/com/example/tasky/feature_login/data/remote/BaseHeaderInterceptor.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.tasky.feature_login.data.remote
+
+import com.example.tasky.BuildConfig
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class BaseHeaderInterceptor : Interceptor {
```

Comment:
If I gave you this `BaseHeaderInterceptor` class name from another project, could you tell me what this is for? 😄 

Probably not, since if a class contains `Base` it almost certainly breaks the single-responsibility principle OR just has an unclear name. If there is a single responsibility, that could be communicated in the class name. In your case, there is, so calling it `ApiKeyInterceptor` would be much clearer and you wouldn't need to open this to know what's inside.